client/packages/programs/src/JsonForms/common/components/CategorizationTabLayout.tsx

```diff
@@ -246,6 +246,7 @@ const UIComponent: FC<LayoutProps & AjvProps> = ({
               textTransform: 'none',
               '& .MuiButton-startIcon': {
                 paddingBottom: '8px',
+                marginLeft: 0,
```

Comment:
I think it needs `marginRight: 0` as well to override this:
![Screenshot 2023-03-15 at 2 50 35 PM](https://user-images.githubusercontent.com/5456533/225183939-57ca58fb-d0ea-4380-afa7-cd3c6bed9610.png)

The "Visit Type" icon looks correct after this. However some of the others don't, but that's because the icons themselves are mis-aligned within their own box:

![Screenshot 2023-03-15 at 2 51 49 PM](https://user-images.githubusercontent.com/5456533/225184085-69ca959f-a4a9-440e-81d7-8cb3daec486c.png)

But we should fix these icons themselves rather than try and mis-align them in the other direction. :) 

We can add a manual `transform` to them to shift the path over within the viewBox, so it looks like this:
![Screenshot 2023-03-15 at 3 07 27 PM](https://user-images.githubusercontent.com/5456533/225186378-96ef3130-8bcb-4d2c-b4eb-8da70e6636dd.png)

That's adding `transform='translate(-75, 0)'` to the CSS data to give:
```
.test-tube {
  mask-image: url("data:image/svg+xml,%3Csvg version='1.1' viewBox='0 0 550 550' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m420 52.5c0-9.625-7.875-17.5-17.5-17.5h-105c-9.625 0-17.5 7.875-17.5 17.5 0 6.4766 3.5 12.074 8.75 15.051v404.95c0 33.773 27.477 61.25 61.25 61.25s61.25-27.477 61.25-61.25v-404.95c5.25-3.0625 8.75-8.5742 8.75-15.051zm-26.25 210h-87.5v-43.75h17.5c4.8125 0 8.75-3.9375 8.75-8.75s-3.9375-8.75-8.75-8.75h-17.5v-17.5h8.75c4.8125 0 8.75-3.9375 8.75-8.75s-3.9375-8.75-8.75-8.75h-8.75v-17.5h17.5c4.8125 0 8.75-3.9375 8.75-8.75s-3.9375-8.75-8.75-8.75h-17.5v-17.5h8.75c4.8125 0 8.75-3.9375 8.75-8.75s-3.9375-8.75-8.75-8.75h-8.75v-26.25h87.5z' transform='translate(-75, 0)'/%3E%3C/svg%3E%0A");
}
```

Looks like several of them could do with a little sliding over ;)

Reply:
[9d6abb4](https://github.com/openmsupply/open-msupply/pull/1358/commits/9d6abb411cec26215ab2b3b2809e6609b012b90b) half the GBV's face disappears when I try to translate it...

Reply:
> [9d6abb4](https://github.com/openmsupply/open-msupply/pull/1358/commits/9d6abb411cec26215ab2b3b2809e6609b012b90b) half the GBV's face disappears when I try to translate it... and the body doesn't move at all

I think the quote nesting is not quite right in that -- `transform='translate(-75, 0)'`should be inside the main `url("... ")` string but outside the path (single-quote).